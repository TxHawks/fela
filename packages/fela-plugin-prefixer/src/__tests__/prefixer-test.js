import prefixer from '../index'

describe('Prefixer plugin', () => {
  it('should prefix styles', () => {
    const style = {
      display: 'flex',
      justifyContent: 'center',
    }

    expect(prefixer()(style)).toMatchSnapshot()
  })

  it('should prefix nested objects', () => {
    const style = {
      display: 'flex',
      ':hover': {
        justifyContent: 'center',
      },
    }

    expect(prefixer()(style)).toMatchSnapshot()
  })
})
