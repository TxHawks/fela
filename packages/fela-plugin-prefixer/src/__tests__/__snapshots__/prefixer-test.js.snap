// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prefixer plugin should prefix nested objects 1`] = `
Object {
  ":hover": Object {
    "WebkitJustifyContent": "center;justify-content:center;-webkit-box-pack:center",
  },
  "display": Array [
    "-webkit-box",
    "-moz-box",
    "-ms-flexbox",
    "-webkit-flex",
    "flex",
  ],
}
`;

exports[`Prefixer plugin should prefix styles 1`] = `
Object {
  "WebkitJustifyContent": "center;justify-content:center;-webkit-box-pack:center",
  "display": Array [
    "-webkit-box",
    "-moz-box",
    "-ms-flexbox",
    "-webkit-flex",
    "flex",
  ],
}
`;
